src/compiler/emitter.ts

```diff
@@ -48,6 +48,21 @@ var __param = (this && this.__param) || function (paramIndex, decorator) {
     return function (target, key) { decorator(target, key, paramIndex); }
 };`;
 
+        const awaiterHelper = `
+var __awaiter = (this && this.__awaiter) || function (generator, ctor) {
+    function resolve(value) { return step(generator.next(value)); }
+    function reject(reason) { return step(generator["throw"](reason)); }

```

Comment:
If someone uses --target ES6 and then babel to get ES5/3 output I don't want it to cause problems with `throw` being a reserved word in ES3 that can't be used as a property name.